unicef/giga-dagster
# feat: add separate lakehouse source path

## What type of PR is this?

feat

## Summary

Allows for the separation of source folders in ADLS for the local dev environment.

## How to test

1. Create a folder in the ADLS container `giga-dataops-dev` named `lakehouse-local-[username]`
2. Copy the source data folders (e.g. raw, bronze, silver, gold, etc.) into the new folder.
3. Set `WAREHOUSE_USERNAME` and `LAKEHOUSE_USERNAME` in `.env` to your username. Make sure this matches with the folder you made.
4. `task`
5. Try using a sensor that interacts with your lakehouse (ex. `migrations__schema_sensor`) and validate that it pulls from the folder you made.
